src/itest/kotlin/nl/lifely/zac/itest/client/KeycloakClient.kt

```diff
@@ -22,16 +22,16 @@ object KeycloakClient {
     private const val ACCESS_TOKEN_ATTRIBUTE = "access_token"
     private const val REFRESH_TOKEN_ATTRIBUTE = "refresh_token"
 
-    fun authenticate() = okHttpClient.newCall(
+    fun authenticate(username: String = "testuser1", password: String = "testuser1") = okHttpClient.newCall(
```

Comment:
Better to use the existing `TEST_USER_1_ID` constant here for the username (and maybe rename it to `TEST_USER_1_USERNAME` come to think of it) and introduce a `TEST_USER_1_PASSWORD` constant in `ItestConfiguration` and use those here.

Reply:
Absolutely right, will do

Reply:
Done